Build failure: manim

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

Build manim

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/312647724

### Relevant log output

```console
(1402 durations < 0.005s hidden.  Use -vv to show these durations.)
=========================== short test summary info ============================
FAILED tests/interface/test_commands.py::test_manim_cfg_subcommand - AssertionError: assert 'Manim Commun...developers.\n' == 'Manim Community v...
= 1 failed, 685 passed, 1 skipped, 5 xfailed, 1 xpassed, 178 warnings in 102.97s (0:01:42) =
```

### Additional context

_No response_

### System metadata

flake from ae814fd3904b621d8ab97418f1d0f2eb0d3716f4

### Notify maintainers



---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/python-modules/manim/failing_tests.nix
+++ b/pkgs/development/python-modules/manim/failing_tests.nix
@@ -89,4 +89,17 @@
   # This tests checks if the manim executable is a python script. In our case it is not.
   # It is a wrapper shell script instead.
   "test_manim_checkhealth_subcommand"
+
+  # failing with:
+  # E       AssertionError: assert 'Manim Commun...developers.\n' == 'Manim Community v0.19.0\n\n'
+  # E
+  # E           Manim Community v0.19.0
+  # E
+  # E         + Usage: manim cfg [OPTIONS] COMMAND [ARGS]...
+  # E         +
+  # E         +   Manages Manim configuration files.
+  # E         + ...
+  # E
+  # E         ...Full output truncated (9 lines hidden), use '-vv' to show
+  "test_manim_cfg_subcommand"
 ]

